internal/controllers/topology/cluster/patches/engine.go

```diff
@@ -290,7 +290,8 @@ func applyPatchesToRequest(ctx context.Context, req *runtimehooksv1.GeneratePatc
 
 		// If a patch doesn't have a corresponding request item, the patch is invalid.
 		if requestItem == nil {
-			return errors.Errorf("unable to find corresponding request item with uid %q for the patch", patch.UID)
+			log.V(5).Infof("Unable to find corresponding request item with uid %q for the patch", patch.UID)
+			return errors.Errorf("unable to find corresponding request item for patch")
```

Comment:
Wondering if we should include a hint that more details can be found in the logs with log level 5?

Regarding log level. Should this be a higher log level?

@fabriziopandini WDYT?

Reply:
Log level 5 is the highest supported according to the logging conventions. I don't think errors / conditions are the right place to be giving direct debug help, unless we do it generically across all errors? It seems like a very general piece of advice, rather than something specific to this issue.

Reply:
(I was suggesting a higher log level given it's an error)

Reply:
Ah - I misunderstood, but I get your point now. I think that's probably a good point - log level 2 is probably the correct level for this.

Reply:
Yup, 2 also sounds fine. Definitely makes it a lot easier to find and I think it's justifiable for an error. 

I think you can also resolve the other conversation. Given that this makes it a lot easier to find the error I think this makes it less important to include the patch in the error.